card/db/migrate_core_cards/data/bootstrap_layout.json

```diff
@@ -54,7 +54,7 @@
cc;\r\n  }  \r\n}\r\n\r\n.card-header > .instruction {\r\n  font-weight: normal;\r\n  font-size: 0.9em;\r\n}\r\n\r\n\r\n/*-------- Special Pages --------*/\r\n\r\n/*-- sign in page --*/\r\n\r\n#sign-in input#login_field,\r\n#sign-in input#password,\r\n#forgot-password input#email {\r\n  width: 50%;\r\n}\r\n#forgot-password {\r\n  margin-top: 40px;\r\n}\r\n\r\n\r\n\r\n.history-view > .card-content,\r\n.conflict-view,\r\nbody#tinymce {\r\n  background: $input_color;\r\n}\r\nbody.mceContentBody {\r\n  margin: 4px;\r\n}\r\n\r\n\r\n/*----------- Change View ---------*/\r\n\r\n.change-view {\r\n  padding: 5px 0px;\r\n  border-bottom: 1px dotted $border_color;\r\n  .change-card{\r\n    font-weight: bold;\r\n  }\r\n  .last-update {\r\n    color: $label_color;\r\n  }\r\n}\r\n\r\n\r\n\r\n/*---------- ( Richtext )-------------*/\r\n\r\n.table-of-contents {\r\n  background: $header_color;\r\n  -moz-border-radius: 10px;\r\n  border-radius: 10px;\r\n  padding: 4px;\r\n  margin: 5px 20px 20px 20px;\r\n  h5 {\r\n    margin: 0 0 0 40px;\r\n  }\r\n  ol {\r\n    list-style-type: upper-roman;\r\n    ol {\r\n      list-style-type: upper-alpha;\r\n      padding-left: 25px\r\n    }\r\n  }\r\n}\r\n\r\n\r\n/* CARDTYPE_SPECIFIC ------------------------------*/\r\n\r\n\r\n.TYPE-toggle.card-content {\r\n  font-style: italic;\r\n}\r\n\r\n.TYPE-image.card-content img {\r\n  max-width: 100%;\r\n}\r\n\r\n/*---------( Search ) ---------- */\r\n\r\n.search-keyword {\r\n  font-style: italic;\r\n}\r\n\r\n.search-count {\r\n  float: right;\r\n  color: $label_color;\r\n}\r\n\r\n\r\n/*----------- (Account Request)--------------*/\r\n\r\n.invite-links {\r\n  padding: 10px;\r\n  margin-top: 10px;\r\n  background: $header_color;\r\n  text-align: center;\r\n}\r\n.invite-links a {\r\n  font-weight: bold;\r\n  margin: 0 20px;\r\n}\r\n\r\n\r\n/*---------- comments -------------------------------------- */\r\n\r\n.comment-box {\r\n  padding: 8px;\r\n  margin: 0px;\r\n  input {\r\n    padding: 0px;\r\n  }\r\n  textarea {\r\n    margin-bottom:5px;\r\n  }\r\n}\r\n.w-comment-author {\r\n  font-style: italic;\r\n  margin-left: 10px;\r\n}\r\n\r\n\r\n/* the logo and credit will not be in the sidebar in most new layouts */\r\n\r\naside {\r\n  > div {\r\n    > a > .SELF-Xlogo.content-view,\r\n    > .SELF-Xcredit.content-view {\r\n      display: block;\r\n      text-align: center;\r\n      padding: 10px;\r\n      background: #fff;\r\n      border: 1px solid #bbb;\r\n      -moz-border-radius: 8px;\r\n      border-radius: 8px;\r\n      img {\r\n        max-width: 100%;\r\n      }\r\n      a {\r\n        color: #186886; /* NOTGREY - Wagn blue */\r\n      }\r\n    }\r\n  }  \r\n}\r\n\r\n\r\n\r\n.closed-content {\r\n  font-size: 1em !important;\r\n}\r\n\r\n\r\n/* english content in css.  ugh */\r\n.search-no-results:after {\r\n  content:'0 results';\r\n}\r\n.closed-content .search-no-results:after {\r\n  content:'';\r\n}\r\n\r\nlegend {\r\n  h2 {\r\n    font-size: 0.7em;\r\n  }\r\n  border: 0;\r\n}\r\n\r\n"
```

Comment:
a lot of the not-yet-submitted changes are to this card (which is good, because it only affects bootstrap layouts)